lib/t/hive_tarkloaderreport.t

```diff
@@ -0,0 +1,100 @@
+=head1 LICENSE
+
+See the NOTICE file distributed with this work for additional information
+   regarding copyright ownership.
+   Licensed under the Apache License, Version 2.0 (the "License");
+   you may not use this file except in compliance with the License.
+   You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+   Unless required by applicable law or agreed to in writing, software
+   distributed under the License is distributed on an "AS IS" BASIS,
+   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+   See the License for the specific language governing permissions and
+   limitations under the License.
+
+=cut
+
+use strict;
+use warnings;
+
+use Log::Log4perl qw(:easy);
+Log::Log4perl->easy_init($DEBUG);
+
+use Cwd;
+use Test::More;
+
+use Data::Dumper;
+
+use Bio::EnsEMBL::Tark::Test::TestDB;
+use Bio::EnsEMBL::Test::MultiTestDB;
+
+use Bio::EnsEMBL::Hive::DBSQL::DBConnection;
+use Bio::EnsEMBL::Hive::Utils::Test qw(standaloneJob get_test_url_or_die make_new_db_from_sqls run_sql_on_db);
+
+# plan tests => 9;
```

Comment:
I generally find plans a burden

Reply:
Will remove as unused. Same for other entries like this.